src/transformers/models/deformable_detr/load_custom.py

```diff
@@ -0,0 +1,35 @@
+import os
+
+
+def load_cuda_kernels():
+    from torch.utils.cpp_extension import load
+
+    root = os.path.join(os.path.dirname(os.path.realpath(__file__)), "custom_kernel")
+    src_files = [
+        os.path.join(root, filename)
+        for filename in [
+            "vision.cpp",
+            os.path.join("cpu", "ms_deform_attn_cpu.cpp"),
+            os.path.join("cuda", "ms_deform_attn_cuda.cu"),
+        ]
+    ]
+
+    load(
+        "MultiScaleDeformableAttention",
+        src_files,
+        # verbose=True,
+        with_cuda=True,
+        extra_include_paths=[root],
+        # build_directory=os.path.dirname(os.path.realpath(__file__)),
```

Comment:
With this commented out, the build won't be cached, no?

Reply:
Also pinging @Narsil here

Reply:
I would need to actually check to be 100%  but the build dir is in `/tmp` by default (and it does cache hit, but it does depend on user setup and might recompile after each reboot for instance)

Reply:
So this is fine @Narsil?